d2game/d2player/help/help.go

```diff
@@ -93,8 +88,9 @@ func (h *Overlay) IsOpen() bool {
 }
 
 func (h *Overlay) IsInRect(px, py int) bool {
-	ww, hh := h.closeButton.GetSize()
-	x, y := h.closeButton.GetPosition()
```

Comment:
This was the bug causing `isInActiveMenusRect` to see `false` erroneously for the Help Menu